apps/cms.c

```diff
@@ -316,6 +317,7 @@ int cms_main(int argc, char **argv)
     const char *CAfile = NULL, *CApath = NULL, *CAstore = NULL;
     char *certsoutfile = NULL, *digestname = NULL, *wrapname = NULL;
     int noCAfile = 0, noCApath = 0, noCAstore = 0;
+    char *digesthex = NULL; unsigned char *digestbin = NULL; long digestlen = 0;
```

Comment:
These should be broken onto separate lines.

Reply:
Done.